README

```diff
@@ -5,3 +5,8 @@ PackageKit is a DBUS abstraction layer that allows the session user to manage
 packages in a secure way using a cross-distro, cross-architecture API.
 
 For more information, please see http://www.packagekit.org
+
```

Comment:
This was probably intended to be for your personal fork, not the upstream README.